FirebaseSessions/Tests/Unit/Library/LifecycleNotifications.swift

```diff
@@ -37,18 +37,7 @@ import Dispatch
 #endif // swift(>=5.9)
 
 extension XCTestCase {
-  func postBackgroundedNotification() {
-    // On Catalyst, the notifications can only be called on a the main thread
```

Comment:
Based on test failure, something else specific to Catalyst might matter?

Reply:
This appears to have been a flake based on now passing CI. We've since bumped the Xcode versions so maybe that resolved it.